nlpretext/textloader/textloader.py

```diff
@@ -1,18 +1,37 @@
+# Copyright (C) 2020 Artefact
```

Comment:
I think textloader.py should be at the same level as preprocessor.py, with a folder only for daskloader and pandasloader (hence with a different name). What do you think ?

Reply:
i thought about it but it can be confusing to not group file that serve the same purpose. To keep the syntaxe as it was before i imported TextLoader in the init. It is transparent for the user

Reply:
Actually, daskloader and pandasloader are more like utils functions for your textloader class so they might be located in another folder (because text loader is using these two separately)